docs/install.md

```diff
@@ -89,31 +89,36 @@ Linux versions).
 
 ## Installing via Docker
 
-Option 1 (Basic, 3.13GB):
+Option 1 (3.13GB, 1.14GB Compressed): Base Image without JupyterLab 
```

Comment:
I'd prefer if we kept the original framing and didn't frame the other options in reference to JupyterLab.